.github/workflows/toxicity-check.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Toxicity check'
+on: 
+  issue_comment:
+    types: ['created', 'edited', 'deleted']
+  pull_request_review:
+  pull_request_review_comment:
+  issues:
+    types: ['opened', 'edited', 'deleted', 'closed', 'reopened', 'locked', 'unlocked']
+  pull_request:
+    types: ['opened', 'edited', 'closed', 'reopened', 'locked', 'unlocked']
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  toxicity-check-job:
+    runs-on: ubuntu-latest
+    name: Toxicity check job
+    steps:
+      - name: Toxicity check
+        uses: TheDutchDevil/toxicity-detection-bot@24daca390d3bd54407c59f848a67090eb02aa447
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Josh can review this PR, but as a Fastlane contributor, I will never share this(Fastlane) token to anyone(any 3rd party libs, which is not under Fastlane team)...because of security reason 😇